application-services/custom/cloud-event/res/configuration.yaml

```
@@ -0,0 +1,33 @@
+Writable:
+  LogLevel: "DEBUG"
+  StoreAndForward:
+    Enabled: false
+    RetryInterval: "5m"
+    MaxRetryCount: 10
+  Telemetry:
+    Interval: "0s" # Disabled since require MessageBus Trigger to be in use.
```

`require MessageBus Trigger to be in use` is no longer correct.

```suggestion
    Interval: "0s" # Disabled since not using metrics
```